core/src/test/java/org/apache/stormcrawler/protocol/selenium/ProtocolTest.java

```diff
@@ -51,7 +51,8 @@ public class ProtocolTest extends AbstractProtocolTest {
     private static final Logger LOG = LoggerFactory.getLogger(ProtocolTest.class);
 
     private static final DockerImageName SELENIUM_IMAGE =
-            DockerImageName.parse("selenium/standalone-chrome:120.0");
+            DockerImageName.parse("seleniarm/standalone-chromium:latest")
```

Comment:
Wonder if we can use a fixed tag? Reasoning would be, that "latest" can vary between environments / runs making reproducability difficult.

Reply:
`latest` is `[124.0](https://hub.docker.com/layers/seleniarm/standalone-chromium/124.0/images/sha256-cd64cfbd8aef032ba2695425e7b6077c7b18995cdd9504c4f70c070ea7fba76f?context=explore) atm

Reply:
Thanks for catching, @rzo1 .  I meant to be explicit on a version number but forgot to switch it before committing 🤦 .  It should be addressed now